pkg/loadbalancer/server.go

```diff
@@ -121,7 +121,7 @@ func (s *Server) EnsureLoadBalancer(ctx context.Context, clusterName string, ser
 		klog.V(2).Infof("updating loadbalancer tunnels on userspace")
 		err = s.tunnelManager.setupTunnels(loadBalancerName(clusterName, service))
 		if err != nil {
-			return nil, err
+			klog.V(2).ErrorS(err, "error setting up tunnels")
```

Comment:
```suggestion
			klog.ErrorS(err, "error setting up tunnels")
```

last thing and we are good, I prefer to log this always for less advanced users